src/components/KTabs/KTabs.vue

```diff
@@ -36,7 +53,9 @@
       <slot
         v-if="activeTab === tab.hash"
         :name="tab.hash.replace('#','')"
-      />
+      >
+        <router-view v-if="tab.route !== undefined" />
```

Comment:
**Comment**: Technically, one `<router-view />` _replacing_ the tab content containers would be sufficient, but I decided to stick to the general approach of this component.